### ISSUE TITLE
Restructure PING probe

### ISSUE THREAD

**Issue by sapiderman** [hyperjumptech/monika#1092]
## Description

PING request should follow the general non-http probe format.

Currently the PING request is tied to the http request:
```yaml
probes:
  - id: 'ping_test'
    name: ping_test
    description: requesting icmp ping
    interval: 10 # in seconds
    requests:
      - url: http://google.com
        ping: true
```

Following a more consistent probe format would be something like:
```yaml
probes:
  - id: 'ping_test'
    name: ping_test
    description: requesting icmp ping
    interval: 10 # every 10 seconds
    ping:
      - uri: http://google.com
```

The original ping request should still be supported for a time. But documentations should promote the correct formatting for PING.

**Comment by haricnugraha** [comment]
The correct term suppose to be restructure instead of refactor, since it change the software behavior to accept a new config format

**Comment by sapiderman** [comment]
merged in #1125
